### DISCUSSION TITLE
连接Unifi路由器出现SSHAuthError (all authentication method failed)

### DISCUSSION THREAD

**Discussion by li9633** [lollipopkit/flutter_server_box#238] [Q&A]
自动更新了一下unifi系统，serverbox连接时发生了该错误，在上一个版本的时候还是正常的。
![_6ef892278d2975db2fd67c66eb6c07c5_-2002232086_Screenshot_20231216-202439](https://github.com/lollipopkit/flutter_server_box/assets/67550161/cac21ecf-c022-43bc-aaf2-100ea471307b)
![_2177dc0e295ba428d787b959cbea2f62_-1204309942_Screenshot_20231216-202505](https://github.com/lollipopkit/flutter_server_box/assets/67550161/90c5c7ee-30e7-4108-940c-1af2da2eba47)
电脑上的Xshell，FinalShell 均可正常连接
uname 结果：Linux Dream-Router 4.4.198-ui-mtk #1 SMP PREEMPT Fri Nov 24 21:02:27 CST 2023 aarch64 GNU/Linux

**Comment by lollipopkit** [comment]
上一个版本是哪个？

是否使用了跳板服务器？跳板服务器功能因为存在问题被下线了

**Reply by li9633** [reply]
目前unifi OS 版本是3.2.7
上一个unifi OS 版本是 3.0.17
没有使用跳板服务器
Serverbox版本是最新的
